internal/host/conditions.go

```diff
@@ -36,7 +39,19 @@ func (v *validator) isClusterInstalling(c *validationContext) bool {
 	return swag.StringValue(c.cluster.Status) == models.ClusterStatusInstalling
 }
 
+func (v *validator) isClusterInError(c *validationContext) bool {
```

Comment:
maybe we can just have one function with isClusterInState? That will get cluster and be part of some cluster util?

Reply:
I will give it a try